src/components/UserForm.tsx

```diff
@@ -68,7 +70,7 @@ const UserForm = () => {
           type="submit"
           className="w-full rounded-md bg-blue-400 px-4 py-4 text-white hover:bg-blue-600 focus:outline-none focus:ring focus:ring-blue-300"
         >
-          Get your stats
+        <Link to={`/Dashboard`}>Get your stats</Link>
```

Comment:
Curly braces and backticks are not needed for sting literals. `to="/Dashboard"`

Reply:
It does not work without it